Civi/ActionSchedule/RecipientBuilder.php

```diff
@@ -468,7 +468,7 @@ protected function prepareRepetitionEndFilter($dateField) {
    */
   protected function prepareAddlFilter($contactIdField) {
     $contactAddlFilter = NULL;
-    if ($this->actionSchedule->limit_to !== NULL && !$this->actionSchedule->limit_to /*0*/) {
+    if ($this->actionSchedule->limit_to !== 1 && $this->actionSchedule->limit_to === 3) {
```

Comment:
Howabout instead we just have: if ($this->actionSchedule->limit_to === 3)

No need for: $this->actionSchedule->limit_to !== 1 - redundant

Reply:
We don't want to start introducing wonky code... :smile: